client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/restaurant/RestaurantScreenModel.kt

```diff
@@ -221,16 +221,17 @@ class RestaurantScreenModel(
 
     override fun onClickDeleteRestaurantMenuItem(restaurant: RestaurantUiState.RestaurantDetailsUiState) {
         tryToExecute(
-            {
-                manageRestaurant.deleteRestaurants(
+            { manageRestaurant.deleteRestaurants(
                     Restaurant(
                         id = restaurant.id,
                         name = restaurant.name,
                         ownerUsername = restaurant.ownerUsername,
                         phoneNumber = restaurant.phoneNumber,
                         rating = restaurant.rating,
                         priceLevel = restaurant.priceLevel,
-                        workingHours = restaurant.workingHours.replace(" ", "").split("-").run {
+                        workingHours =
+                        restaurant.workingHours.replace(" ", "")
+                            .split("-").run {
                             Pair(
                                 Time.parseToCustomTime(get(0)),
                                 Time.parseToCustomTime(get(1))
```

Comment:
i think it could be better if we make it mapper function

Reply:
This was changed in another pr